docs/Ways-of-Working/Communication/Documentation-Updates.md

```diff
@@ -0,0 +1,31 @@
+# Documentation Updates
+
+Documentation at AM is an ongoing process, we're currently focused on improving documentation and making it easier for everyone to access and discover information to help their understanding of how we work, and the platforms we work on.
+
+!!! Tip 
+    Documentation is a living document and is the responsibility of the whole department to improve and update the documentation, don't just leave it for the next person to do.
+
+
+We have 3 locations where we store documentation as a department:
```

Comment:
I think this is starting to look like the documentation directory in the KB. Which also highlights that there are more than 3 places, and loads more types of documentation in many other places.

I would suggest that the Playbook expresses the fact that documentation is a vital part of communication and collaboration, and points to the KB documentation directory to understand current tools for each type.

The KB doc directory could then be updated to contain all these things, pointing out their location, purpose, and and contribution guidelines (or signposting to more in depth guidelines)